colin/core/target.py

```diff
@@ -130,6 +131,23 @@ def labels(self):
             return self.instance.labels
         return self.instance.get_metadata()["Config"]["Labels"]
 
+    def get_output(self, cmd):
+        if isinstance(cmd, six.string_types):
+            cmd = [cmd]
+        if self.target_type == TargetType.CONTAINER:
+            if not self.instance.is_running():
+                raise ColinException("Cannot get output for stopped container.")
+            return "".join([o.decode() for o in self.instance.execute(command=cmd)])
+
+        elif self.target_type == TargetType.CONTAINER_IMAGE:
+            container = self.instance.run_via_binary(command=cmd)
```

Comment:
Could we get an integration test here for images which do
```
FROM fedora:28
ENTRYPOINT ["/bin/ls"]
CMD ["/"]
```

because I'm pretty sure that present solution won't work for ^ that mess.